client/blocks/plan-storage/bar.jsx

```diff
@@ -58,7 +58,7 @@ class PlanStorageBar extends Component {
 		return (
 			<div className={ classes }>
 				<ProgressBar
-					className="plan-storage__bar"
+					className="plan-storage__storage-bar"
```

Comment:
I think it's needed: https://github.com/Automattic/wp-calypso/blob/master/client/blocks/plan-storage/style.scss#L5

if we are going to rename the className it should be done in the styles file as well.